libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/TarFetchedCallbackTest.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import android.os.AsyncTask;
+
+import com.mapbox.services.android.navigation.v5.utils.DownloadTask;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Response;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
+public class TarFetchedCallbackTest {
+
+  @Test
+  public void onSuccessfulResponse_downloadTaskIsExecuted() {
+    RouteTileDownloader downloader = mock(RouteTileDownloader.class);
```

Comment:
Could you extract (if possible) some of the shared arrange code across these tests into a `private` factory method?